apps/prairielearn/src/components/QuestionScore.html.ts

```diff
@@ -63,23 +64,35 @@ export function QuestionScorePanel({
               `
             : ''}
           ${assessment.type === 'Homework'
-            ? // Only show previous variants if the question allows multiple variants, or there are multiple variants (i.e., they were allowed at some point)
-              !question.single_variant ||
-              (instance_question_info.previous_variants?.length ?? 0) > 1
-              ? html`
-                  <tr>
-                    <td colspan="2" class="text-wrap">
-                      All variants:
-                      ${QuestionAwardedPoints({
-                        instanceQuestionId: instance_question.id,
-                        previousVariants: instance_question_info.previous_variants,
-                        currentVariantId: variant?.id,
-                        urlPrefix,
-                      })}
-                    </td>
-                  </tr>
-                `
-              : ''
+            ? html`
+                ${assessment_question.max_auto_points
```

Comment:
This condition covers two cases:
* A purely manually grade question;
* A question with no points at all.

I agree with the current functionality (i.e., not showing the value in either case), but I'd prefer to see a short comment that this is being considered.

Reply:
Comment added in 8512f9e.